### PR TITLE
567 windows 11 installer not working

### PR DESCRIPTION
**Pull request by Pierregvx**

### CODE REVIEW

**Review by thomas-senechal — APPROVED** [review]

**Comment by Thykof** [comment]
I don't understand how the changes fix the installation

**Comment by Pierregvx** [comment]
> I don't understand how the changes fix the installation

@thomas-senechal would explain it better than be but the windows decoder was crashing because of certains characters so I added a try catch to rerurn the default encoding if needed

**Comment by thomas-senechal** [comment]
> > I don't understand how the changes fix the installation
> 
> @thomas-senechal would explain it better than be but the windows decoder was crashing because of certains characters so I added a try catch to reyurn the default encoding if needed

https://github.com/massalabs/thyra/issues/567#issuecomment-1488389217 Some context

File: scripts/installer.py [hunk 1]
```diff
@@ -58,14 +59,17 @@ def executeCommand(self, command, shell=False, allow_failure=False) -> tuple[str
...
 
-            if allow_failure == False and process.returncode != 0:
-                self.printErrorAndExit(f"Command failed with error : {stderr.decode(locale.getpreferredencoding(False))}")
+                if allow_failure == False and process.returncode != 0:
+                    self.printErrorAndExit(f"Command failed with error : {stderr.decode(locale.getpreferredencoding(False))}")
 
-            return (stdout.decode(locale.getpreferredencoding(False)), stderr.decode(locale.getpreferredencoding(False)))
+                return (stdout.decode(locale.getpreferredencoding(False)), stderr.decode(locale.getpreferredencoding(False)))
+            except :
+                return(stdout, stderr)
```
Comment (Thykof) [inline scripts/installer.py:72]: the bug is in the decoding of std out and str err?

Comment (Pierregvx) [reply scripts/installer.py:72]: it was just for stderr in this case

**Review by Thykof — DISMISSED** [review]

**Review by Thykof — APPROVED** [review]
LGTM

**Comment by Thykof** [comment]
CI is red because testnet is down, this PR doesn't modify API. let's merge

**Pull request merged**